src/Features/Core/Diagnostics/EngineV1/DiagnosticIncrementalAnalyzer.cs

```diff
@@ -24,997 +24,998 @@ namespace Microsoft.CodeAnalysis.Diagnostics.EngineV1
 
     internal partial class DiagnosticIncrementalAnalyzer : BaseDiagnosticIncrementalAnalyzer
     {
-        private static readonly int s_stateTypeCount = Enum.GetNames(typeof(StateType)).Count();
-        private static readonly ImmutableArray<StateType> s_documentScopeStateTypes = ImmutableArray.Create<StateType>(StateType.Syntax, StateType.Document);
+            private static readonly int s_stateTypeCount = Enum.GetNames(typeof(StateType)).Count();
+            private static readonly ImmutableArray<StateType> s_documentScopeStateTypes = ImmutableArray.Create<StateType>(StateType.Syntax, StateType.Document);
```

Comment:
That file had merge conflict due to rename.


-          Heejae

From: Srivatsn Narayanan [mailto:notifications@github.com]
Sent: Monday, February 16, 2015 6:38 PM
To: dotnet/roslyn
Cc: HeeJae Chang
Subject: Re: [roslyn] add property bag to diagnostic and remember origin of the diagnostic (#510)


In src/Features/Core/Diagnostics/EngineV1/DiagnosticIncrementalAnalyzer.cs<https://github.com/dotnet/roslyn/pull/510#discussion_r24789127>:

> @@ -24,997 +24,998 @@ namespace Microsoft.CodeAnalysis.Diagnostics.EngineV1

>

>      internal partial class DiagnosticIncrementalAnalyzer : BaseDiagnosticIncrementalAnalyzer

>      {

> -        private static readonly int s_stateTypeCount = Enum.GetNames(typeof(StateType)).Count();

> -        private static readonly ImmutableArray<StateType> s_documentScopeStateTypes = ImmutableArray.Create<StateType>(StateType.Syntax, StateType.Document);

> +            private static readonly int s_stateTypeCount = Enum.GetNames(typeof(StateType)).Count();

> +            private static readonly ImmutableArray<StateType> s_documentScopeStateTypes = ImmutableArray.Create<StateType>(StateType.Syntax, StateType.Document);

The formatting is making the diff hard to understand - is there any change in this file at all?

—
Reply to this email directly or view it on GitHub<https://github.com/dotnet/roslyn/pull/510/files#r24789127>.